docs/learn/data-submitters/submission-guide/ingesting-data.mdx

```diff
@@ -3,47 +3,35 @@ breadcrumbs:
-4. **Validation** (done by AnVIL ingestion team)
-
-## Step-by-Step Instructions
-For details, see [How to stage data in your AnVIL deposit workspace](https://support.terra.bio/hc/en-us/articles/28970864241435-How-to-stage-data-in-an-AnVIL-deposit-workspace).
+### 1. Log into AnVIL
+You will use a Google ID for SSO to access your assigned data deposit workspace on [anvil.terra.bio](https://anvil.terra.bio). Note that an institutional email is required for login to access controlled-data.     
 
-## Next steps (done by ingestion team)
-After you stage the data in the deposit workspace, the AnVIL team will perform these pre-ingestion operations.
+### 2. Set up your workspace cloud storage 
+To facilitate ingestion into TDR, the workspace cloud storage must have a particular directory structure.      
 
-### Validation (automated)
-These tests will be executed once data has been ported to AnVIL.
+### 3. Upload data object files to the deposit workspace storage (optional)           
+You’ll import all files to the Uploads folder in the submission workspace using gcloud storage command line tool (recommended) or tool of your choice. Note that if your object files are already stored in Google Cloud Storage, you can skip this step.       
 
-- **QC check of submission form to genomic object files**
-Check that the number of files match the number in the submission Google form
+### 4. Verify md5 hash for all data object files         
+You can do this by running the **CreateWorkspaceFileManifest** workflow (included in the deposit workspace) or examining the GCS metadata directly.      
 
-- **QC check of phenotype and metadata**
-Make sure the phenotype file data fields match the defined data model and sample IDs are consistent with phenotype and linked to a subject and consent.
+### 5. Upload all tables (TSV load files)** from the data model         
+You’ll create tables in the submission workspace by importing the TSV files using the Data Uploader (recommended) or the Terra UX.       
 
-- **Ingestion Validation (automated)**
-To confirm the ingested data transferred as expected and maintain the file integrity, Google automatically checks the md5 sum of the end file against the original after each file transfer.
+### 6. **Validate data**     
+Once the data object files and tabular data are staged in the submission workspace, you’ll run a QC workflow to validate the data. 
 
 
-### Data Indexing (genomic object files)
-
-Once in the workspace buckets, object files, such as sequencing data, are indexed with a global unique ID (GUID). This allows access across AnVIL tools without requiring copies to be created and transferred across environments.
+## Step-by-Step Instructions
+For details, see [How to stage data in your AnVIL deposit workspace](https://support.terra.bio/hc/en-us/articles/28970864241435-How-to-stage-data-in-an-AnVIL-deposit-workspace).
```

Comment:
@acliffe38 the link to "How to stage data in your AnVIL deposit workspace" doesn't appear to work for me?
https://support.terra.bio/hc/en-us/articles/28970864241435-How-to-stage-data-in-an-AnVIL-deposit-workspace

Reply:
Awesome thank you!